corehq/apps/domain/forms.py

```diff
@@ -1129,6 +1163,28 @@ def clean(self):
                    "to send an email to the partner.").format(dimagi_user.username)
             self.add_error('dimagi_contact', msg)
 
+    def save_gsheet_limit(self, domain, limit_value):
+        try:
+            limit = ProjectLimit.objects.get(domain=domain)
```

Comment:
shouldn't this also be filtered by the google sheets limit type? in the future there will be other limits supported